packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -192,6 +192,7 @@ export const messages = {
   REST_API_description: 'Execute the query with REST API',
   tooling_API_description: 'Execute the query with Tooling API',
   telemetry_legal_dialog_message:
-    'You agree that Salesforce Extensions for VS Code may collect usage information, user environment, and crash reports for product improvements. Learn how to opt out.',
+    'You agree that Salesforce Extensions for VS Code may collect usage information, user environment, and crash reports for product improvements. Learn how to ' +
```

Comment:
Why do we have this as two strings ? `'You agree ... Learn how to ' + '[opt out]...'`

Reply:
Good point